packages/cli/src/oclif/commands/invoke.js

```diff
@@ -0,0 +1,685 @@
+          let inputStream;
+          if (filePath === '-') {
+            inputStream = process.stdin;
+          } else {
+            inputStream = fs.createReadStream(filePath, { encoding: 'utf8' });
+          }
+          inputData = await readStream(inputStream);
+        }
+        inputData = JSON.parse(inputData);
+      } else {
+        inputData = {};
+      }
+
+      const { timezone } = this.flags;
+      const meta = {
+        isLoadingSample: this.flags.isLoadingSample,
+        isFillingDynamicDropdown: this.flags.isFillingDynamicDropdown,
+        isPopulatingDedupe: this.flags.isPopulatingDedupe,
+        limit: this.flags.limit,
+        page: this.flags.page,
+        isTestingAuth: false, // legacy property
+      };
+      const output = await this.invokeAction(
+        appDefinition,
+        actionTypePlural,
+        action,
+        inputData,
+        authData,
+        meta,
+        timezone,
+        zcacheTestObj
+      );
+      console.log(JSON.stringify(output, null, 2));
+    }
+  }
+}
+
+InvokeCommand.flags = buildFlags({
+  commandFlags: {
+    inputData: flags.string({
+      char: 'i',
+      description:
+        'The input data to pass to the action. Must be a JSON-encoded object. The data can be passed from the command directly like \'{"key": "value"}\', read from a file like @file.json, or read from stdin like @-.',
+    }),
+    isLoadingSample: flags.boolean({
+      description:
+        'Set bundle.meta.isLoadingSample to true. When true in production, this run is initiated by the user in the Zap editor trying to pull a sample.',
+      default: false,
+    }),
+    isFillingDynamicDropdown: flags.boolean({
+      description:
+        'Set bundle.meta.isFillingDynamicDropdown to true. Only makes sense for a polling trigger. When true in production, this poll is being used to populate a dynamic dropdown.',
+      default: false,
+    }),
+    isPopulatingDedupe: flags.boolean({
+      description:
+        'Set bundle.meta.isPopulatingDedupe to true. Only makes sense for a polling trigger. When true in production, the results of this poll will be used initialize the deduplication list rather than trigger a Zap. This happens when a user enables a Zap.',
+      default: false,
+    }),
+    limit: flags.integer({
+      description:
+        'Set bundle.meta.limit. Only makes sense for a trigger. When used in production, this indicates the number of items you should fetch. -1 means no limit.',
+      default: -1,
+    }),
+    page: flags.integer({
+      char: 'p',
+      description:
+        'Set bundle.meta.page. Only makes sense for a trigger. When used in production, this indicates which page of items you should fetch. First page is 0.',
+      default: 0,
+    }),
+    timezone: flags.string({
+      char: 'z',
+      description:
+        'Set the default timezone for datetime fields. If not set, defaults to America/Chicago, which matches Zapier production behavior.',
```

Comment:
I think people can infer from the timezone string `American/Chicago` but would it be helpful to provide a link for options? 

i.e.: https://data.iana.org/time-zones/tzdb-2021a/zone1970.tab

Reply:
Good call! Added the link in 10ff7f7. I use the wikipedia link because IANA.org has [releases](https://data.iana.org/time-zones/) over time and the latest link can change.